showcase/app/styles/mock-components/demo/breakpoints/ruler.scss

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+@use "sass:map";
+// from @hashicorp/design-system-components package
+// @use "mixins/breakpoints" as *;
```

Comment:
@aklkv @RobbieTheWagner can you help me understand why this is not working? 

<img width="589" alt="screenshot_4936" src="https://github.com/user-attachments/assets/e68cfb7c-b75f-407c-88ae-0fe56a8d37b4" />

The same approach is used in Cloud UI ([see here](https://github.com/hashicorp/cloud-ui/blob/main/addons/core/package/src/styles/core/_overrides.scss#L11-L12)) so it should work here too. The only difference I can think of is that Cloud UI is using rollup while the showcase doesn't.

In any case, any idea how to get this working?

Reply:
this should work, maybe you have not rebuild components package?

Reply:
I have confirmed it just works

Reply:
@aklkv I still see the same error, even after pulling in your comit:
<img width="1190" alt="screenshot_4939" src="https://github.com/user-attachments/assets/d218f4a1-a790-4f85-9543-21ee69c5d8e1" />

Reply:
since test are passing makes me think that there is something wrong with you local setup, try deleting dist folder in components package and re build it

Reply:
nvm, I think I got it reproduced locally, I will check what is up

Reply:
I suspect is something related to caching:
<img width="1288" alt="screenshot_4940" src="https://github.com/user-attachments/assets/5dec514c-c8c4-4df9-8f32-47221e929d75" />

This is after I moved the brekpoints file from `foundations/` to `utils`, and yes I have deleted `dist`, and re-run `pnpm build` and/or `pnpm start` under the `packages/components`

Reply:
`pnpm i` the issues

Reply:
```
npx npkill -D -x -y
pnpm i
pnpm start
```

same error
<img width="1194" alt="screenshot_4941" src="https://github.com/user-attachments/assets/3f2e23ef-3017-46db-8ceb-016a6b317971" />

Reply:
yes, `pnpm` sync dependancies on `pnpm install`, where as before it was symlink it. I am thinking maybe we should disable this behaviour. @RobbieTheWagner any thoughts

Reply:
I have confirmed that for existing files this strategy works fine they are being correctly linked and changes are propagated correctly but in cases where you rename/remove the file you need to reestablish the link by `pnpm i`. and this is what you are seeing after moving the file our as well

Reply:
@RobbieTheWagner can you help me on this? I am kind of stuck.

Reply:
is there other issues you are facing after changing file location and re running `pnpm i`?